dc-cudami-editor/src/components/modals/imageAdder/ImageSelector.jsx

```diff
@@ -70,9 +70,16 @@ class ImageSelector extends Component {
       showUploadSuccess: true,
     })
     setTimeout(() => this.setState({showUploadSuccess: false}), 3000)
+    const subMimeType = responseJson.mimeType.split('/')[1]
+    const mimeExtensionMapping = {
```

Comment:
Nachdem das ja zwischen den zwei Komponenten geteilt ist, wäre das vllt besser in einem Modul untergebracht das dann importiert wird.

Reply:
Ist jetzt in `utils.js`.